diff --git a/ps2xRuntime/src/lib/Kernel/Syscalls/Dispatcher.cpp b/ps2xRuntime/src/lib/Kernel/Syscalls/Dispatcher.cpp
index 3b2be31..65ab67d 100644
--- a/ps2xRuntime/src/lib/Kernel/Syscalls/Dispatcher.cpp
+++ b/ps2xRuntime/src/lib/Kernel/Syscalls/Dispatcher.cpp
@@ -292,6 +292,15 @@ namespace ps2_syscalls
         case static_cast<uint32_t>(-0x78):
             ps2_stubs::sceSifSetDChain(rdram, ctx, runtime);
             return true;
+        case 0x79:
+            ps2_stubs::sceSifSetReg(rdram, ctx, runtime);
+            return true;
+        case 0x7A:
+            ps2_stubs::sceSifGetReg(rdram, ctx, runtime);
+            return true;
+        case 0x7B:
+            ps2_stubs::sceSifStopDma(rdram, ctx, runtime);
+            return true;
         case 0x7F:
             GetMemorySize(rdram, ctx, runtime);
             return true;
